sources/assets/exegol/load_supported_setups.sh

```diff
@@ -175,48 +197,39 @@ function deploy_bloodhound_customqueries_merge() {
 }
 
 function deploy_bloodhound_customqueries_replacement() {
-  # Replace Exegol's customqueries.json file with the merge of ones from my-resources
+  colorecho "Merging BloodhHound User Custom Queries with Exegol Custom Queries"
```

Comment:
It's now wrong here. This function is not meant to merge with Exegol's queries, but to merge Custom Queries and then replace Exegol's

Reply:
Or, I could've [RTFM](https://exegol.readthedocs.io/en/dev-images/exegol-image/my-resources.html#bloodhound-customqueries-config) and get my answer
This will merge all queries in the replacement dir and produce a file that will replace exegol's custom queries

Reply:
Better now? (https://github.com/ThePorgs/Exegol-images/pull/246/commits/e3c3918583638c1a75110ebc768e1e63d91e5c42)

Reply:
Yes :)